frontend/src/components/AgentViewArtifactViewer.css

```
@@ -0,0 +1,205 @@
+}
+
+.agent-view-artifact-viewer__source-list li {
+  display: inline-flex;
+  align-items: baseline;
+  gap: 4px;
+}
+
+.agent-view-artifact-viewer__source-link {
+  color: var(--status-info);
+  text-decoration: none;
+  border-bottom: 1px solid currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)